Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -2239,6 +2239,69 @@ PatchLegacyCommpage (
   return EFI_NOT_FOUND;
 }
 
+STATIC
+CONST UINT8
+  mAppleVTDPatchFindCaseySJ[] = {
+  0x4C, 0x89, 0xF6, 0xE8,
```

Comment:
I suggest masking the call value out, by finding E8 xx xx xx xx (mask FF 00 00 00 00).

Reply:
Will do.

Reply:
Good suggestion to use Find and Replace masks. This has been applied and tested.

Reply:
Agreed -- modified and verified.